### PR TITLE
#4169 Write tests for two methods and their private methods

### PR DESCRIPTION
**Pull request by MaksymKorolkov**
Tests for getAllCouriers() and setLimitDescription() methods

### CODE REVIEW

**Review by danylohlynsky — APPROVED** [review]

**Review by Andrii-Yezenitskyi — APPROVED** [review]

File: service/src/test/java/greencity/service/SuperAdminServiceImplTest.java [hunk 1]
```diff
@@ -110,6 +110,52 @@ void deleteTariffServiceTest() {
...
 
+    @Test
+    void setLimitDescription() {
+        Courier courier = ModelUtils.getCourier();
+        CourierTranslation courierTranslation = CourierTranslation.builder()
+            .id(1L)
+            .name("Test")
+            .language(getLanguage())
+            .courier(courier)
+            .build();
```
Comment (VadymMakitra) [inline service/src/test/java/greencity/service/SuperAdminServiceImplTest.java]: Move the building object to the ModelUtils class pls.

Comment (VadymMakitra) [inline service/src/test/java/greencity/service/SuperAdminServiceImplTest.java:121]: Move the building object to the ModelUtils class pls.

File: service/src/test/java/greencity/service/SuperAdminServiceImplTest.java [hunk 2]
```diff
@@ -110,6 +110,52 @@ void deleteTariffServiceTest() {
...
+        when(courierRepository.findById(1L)).thenReturn(Optional.of(courier));
+        when(courierTranslationRepository.findCourierTranslationByCourier(getCourier()))
+            .thenReturn(courierTranslation);
+        when(courierTranslationRepository.save(courierTranslation)).thenReturn(courierTranslation);
+
+        GetCourierTranslationsDto courierTranslationsDto = GetCourierTranslationsDto.builder()
+            .id(1L)
+            .languageCode("ua")
+            .name("Test")
+            .build();
```
Comment (VadymMakitra) [inline service/src/test/java/greencity/service/SuperAdminServiceImplTest.java]: Move the building object to the ModelUtils class pls.

Comment (VadymMakitra) [inline service/src/test/java/greencity/service/SuperAdminServiceImplTest.java:132]: Move the building object to the ModelUtils class pls.

File: service/src/test/java/greencity/service/SuperAdminServiceImplTest.java [hunk 3]
```diff
@@ -110,6 +110,52 @@ void deleteTariffServiceTest() {
...
+        verify(courierTranslationRepository).save(courierTranslation);
+    }
+
+    @Test
+    void getAllCouriersTest() {
+        CourierDto courierDto = CourierDto.builder()
+            .courierId(1L)
+            .courierStatus("ACTIVE")
+            .courierTranslationDtos(getCourierTranslationDtoList())
+            .build();
```
Comment (VadymMakitra) [inline service/src/test/java/greencity/service/SuperAdminServiceImplTest.java]: Move the building object to the ModelUtils class pls.

Comment (VadymMakitra) [inline service/src/test/java/greencity/service/SuperAdminServiceImplTest.java:147]: Move the building object to the ModelUtils class pls.

**Review by NataliaHlazova — APPROVED** [review]

**Pull request merged**